### PR TITLE
update `task-generated mapping` example

### PR DESCRIPTION
**Pull request by aspain**
current example results in `TypeError: 'int' object is not iterable` for mapped tasks 0 & 1. Updating example with values that can be used with list()

---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in a newsfragement file, named `{pr_number}.significant.rst`, in [newsfragments](https://github.com/apache/airflow/tree/main/newsfragments).

### CODE REVIEW

File: docs/apache-airflow/concepts/dynamic-task-mapping.rst [hunk 1]
```diff
@@ -152,7 +152,7 @@ Up until now the examples we've shown could all be achieved with a ``for`` loop
     def make_list():
         # This can also be from an API call, checking a database, -- almost anything you like, as long as the
         # resulting list/dictionary can be stored in the current XCom backend.
-        return [1, 2, {"a": "b"}, "str"]
+        return [[1, 2], ("a", "b") , {"a": "b"}, "str"]
```
Comment (jhtimmins) [inline docs/apache-airflow/concepts/dynamic-task-mapping.rst]:
```suggestion
        return [[1, 2], ("a", "b"), {"a": "b"}, "str"]
```

The extra space is causing static tests to fail.

**Review by jhtimmins — DISMISSED** [review]
Can you point to where this is causing test failure?

**Comment by aspain** [comment]
🤦 thanks @jhtimmins sorry about that

**Review by uranusjr — COMMENTED** [review]
This is wrong, the example should work. Are you incorrectly using `make_list` against the `BaseOperator` example below?

**Comment by aspain** [comment]
I copied the example DAG as-is from the [Task-generated Mapping](https://airflow.apache.org/docs/apache-airflow/2.3.0/concepts/dynamic-task-mapping.html#task-generated-mapping) section which results in the first 2 mapped tasks failing:
image

Task log error:

```
[2022-05-02, 22:20:26 UTC] {taskinstance.py:1888} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.9/site-packages/airflow/decorators/base.py", line 179, in execute
    return_value = super().execute(context)
  File "/usr/local/lib/python3.9/site-packages/airflow/operators/python.py", line 171, in execute
    return_value = self.execute_callable()
  File "/usr/local/lib/python3.9/site-packages/airflow/operators/python.py", line 189, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
  File "/usr/local/airflow/dags/dyn2.py", line 15, in consumer
    print(list(arg))
TypeError: 'int' object is not iterable
[2022-05-02, 22:20:26 UTC] {taskinstance.py:1394} INFO - Marking task as FAILED. dag_id=dynamic-map2, task_id=consumer, map_index=0, execution_date=20220412T000000, start_date=20220502T222026, end_date=20220502T222026
[2022-05-02, 22:20:26 UTC] {standard_task_runner.py:92} ERROR - Failed to execute job 909 for task consumer ('int' object is not iterable; 390)
[2022-05-02, 22:20:26 UTC] {local_task_job.py:156} INFO - Task exited with return code 1
```

**Comment by uranusjr** [comment]
Ah, I know where the problem is. `make_list` is fine, but `consumer` should not do `list(arg)`. We should change it to simply `print(arg)`.

**Review by uranusjr — APPROVED** [review]

**Review by aspain — COMMENTED** [review]
ended up changing `print(list(arg))` to `print(arg)` and keeping the original return from the example

**Comment by uranusjr** [comment]
I’m just going to merge this since it’s a one-line documentation change that I should have gotten right in the first place.

**Pull request merged**